template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/MSTest-CSharp/.template.config/template.json

```
@@ -42,6 +42,12 @@
       "defaultValue": "false",
       "description": "Whether or not to enable packaging (via \"dotnet pack\") for the project."
     },
+    "EnableRunner": {
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false",
+      "description": "Whether or not to enable MSTest runner for the project. More information at https://aka.ms/mstest/runner."
```

```suggestion
      "description": "Whether or not to enable MSTest runner for the project. More information at https://aka.ms/mstest/runner"
```

Not grammatically correct, but makes copying the link easier if you can't simply click it in your commandline.